### DISCUSSION TITLE
what's the principle of the integration of snATAC-seq and scRNA-seq data during label transfer step?

### DISCUSSION THREAD

**Discussion by Melody-cell** [aertslab/pycisTopic#192] [General]
I am confused how this step can integrate ATAC and RNA-seq?
![image](https://github.com/user-attachments/assets/ffbbb76d-9b29-4eda-ad90-80fb97163269)
And if my ATAC's cell types are not corresponding to RNA data's cell types, can I use this step: "plot a confusion matrix" ?

**Comment by SeppeDeWinter** [comment]
Hi @Melody-cell 

About which part are you confused?
This function uses the listed methods (harmony etc). to integrate the RNA-count matrix with the gene-activity matrix (calculated from chromatin accessibility).

If your cell types don't match you won't be able to draw a confusion matrix. However, I'm not sure why you would want to do this?

All the best,

Seppe

**Reply by Melody-cell** [reply]
Hi, @SeppeDeWinter ,
I'm going to use unpaired snATAC-seq data and scRNA-seq data from different articles to run scenicplus, i saw this step (label transfer) can integrate 2 datasets (ATAC and RNA) together, right?

**Reply by SeppeDeWinter** [reply]
That's correct.

**Reply by Melody-cell** [reply]
But my ATAC data's cell type and RNA data's cell type aren't completely the same, they didn't match very well (cause they come from different articles), does this make sense ?

**Reply by SeppeDeWinter** [reply]
Of that I'm not sure!
You'll have to be the judge wether it makes biological sense to integrate both datasets 🙂 

All the best,

Seppe

**Reply by Melody-cell** [reply]
OK I see,
thank you very much for your patient reply!
